doc/manuals/devel/cookbook.md

```diff
@@ -106,16 +106,15 @@ Note the right-most column saying `(environment variable)` for the `-fg` option.
 [Top](#top)
 
 ## Adding a REST service
-The REST services that the Orion Context Broker supports are items in the `RestService` vector `restServiceV`, found in [the main program](sourceCode.md#srcappcontextbroker) in `contextBroker.cpp`. This vector is a reminiscent from back when Orion also implemented the FIWARE Configuration Manager and thus had to be able to assume different roles. The set of services that are supported pretty much defines the role and by starting the REST interface with one `RestService` vector or another took care of the role selection. Now Orion has only one role, to work as context broker, but the `RestService` vector remains.
+The REST services that the Orion Context Broker supports are items in the seven `RestService` vectors `restServiceV`, found in [](sourceCode.md#srcappcontextbroker) in `orionRestServices.cpp`. There are one service vector per HTTP Method/Verb that Orion supports: GET, PUT, POST, PATCH, DELETE, OPTIONS, plus a special vector for 'bad verb'. The set of services that are supported pretty much defines the role and by starting the REST interface with one `RestService` vector or another defines what the broker is able to do, all its services are included in these seven vectors.
```

Comment:
The `[]` is empty... It should be the text for the link.

Reply:
Fixed in d65b3f9